<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>621</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="title">
       <property name="font">
        <font>
         <family>Academy Engraved LET</family>
         <pointsize>64</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Graphical Phonebook</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="saveAndLoadBox">
       <item>
        <widget class="QTabWidget" name="tabWidgetBox">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tabWidgetAdd">
          <attribute name="title">
           <string>ADD CONTACT</string>
          </attribute>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>30</y>
             <width>286</width>
             <height>269</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="addContactMain">
              <item>
               <layout class="QVBoxLayout" name="addContactWelcomeLayoutOuter">
                <item>
                 <widget class="QLabel" name="welcome1">
                  <property name="font">
                   <font>
                    <family>Academy Engraved LET</family>
                    <pointsize>24</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Welcome!</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="welcomeLayoutInner">
                  <item>
                   <widget class="QLabel" name="welcome2">
                    <property name="font">
                     <font>
                      <family>Nanum Myeongjo</family>
                     </font>
                    </property>
                    <property name="text">
                     <string>If you want to add information to Contact, </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="welcome3">
                    <property name="font">
                     <font>
                      <family>Nanum Myeongjo</family>
                     </font>
                    </property>
                    <property name="text">
                     <string>please fill the blanks below:</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="addContactContents">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <item>
                   <layout class="QHBoxLayout" name="name">
                    <item>
                     <widget class="QLabel" name="nameLabel">
                      <property name="layoutDirection">
                       <enum>Qt::RightToLeft</enum>
                      </property>
                      <property name="text">
                       <string>Name</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="nameInput">
                      <property name="dragEnabled">
                       <bool>true</bool>
                      </property>
                      <property name="clearButtonEnabled">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="phoneNumber">
                    <item>
                     <widget class="QLabel" name="phoneNumberLabel">
                      <property name="layoutDirection">
                       <enum>Qt::RightToLeft</enum>
                      </property>
                      <property name="text">
                       <string>PhoneNumber</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="phoneNumberInput">
                      <property name="dragEnabled">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="nickname">
                    <item>
                     <widget class="QLabel" name="nicknameLabel">
                      <property name="layoutDirection">
                       <enum>Qt::RightToLeft</enum>
                      </property>
                      <property name="text">
                       <string>Nickname</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="nickNameInput">
                      <property name="dragEnabled">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QRadioButton" name="checkBookmarkRadioButton">
              <property name="text">
               <string>Check Bookmark</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="confirmButton">
              <property name="text">
               <string>Confirm</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tabWidgetSearch">
          <attribute name="title">
           <string>SEARCH</string>
          </attribute>
          <widget class="QLabel" name="widgetSearchLabel">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>30</y>
             <width>181</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>AppleMyungjo</family>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Search Contacts</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="widgetSearchTextEditBar">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>70</y>
             <width>311</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QComboBox" name="widgetSearchComboBox">
           <property name="geometry">
            <rect>
             <x>470</x>
             <y>10</y>
             <width>103</width>
             <height>32</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>name</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>phoneNumber</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>nickName</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bookMark</string>
            </property>
           </item>
          </widget>
          <widget class="QPushButton" name="widgetSearchButton">
           <property name="geometry">
            <rect>
             <x>480</x>
             <y>40</y>
             <width>61</width>
             <height>32</height>
            </rect>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
          <widget class="QLabel" name="widgetSearchTODO">
           <property name="geometry">
            <rect>
             <x>400</x>
             <y>310</y>
             <width>201</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>need to make delete button here</string>
           </property>
          </widget>
          <widget class="QTableWidget" name="tableWidgetSearch">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>130</y>
             <width>461</width>
             <height>171</height>
            </rect>
           </property>
           <column>
            <property name="text">
             <string>Bookmark</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tel.</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Nickname</string>
            </property>
           </column>
          </widget>
          <widget class="QPushButton" name="pushButtonReset">
           <property name="geometry">
            <rect>
             <x>480</x>
             <y>70</y>
             <width>100</width>
             <height>32</height>
            </rect>
           </property>
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tabWidgetShow">
          <attribute name="title">
           <string>SHOW</string>
          </attribute>
          <widget class="QTableView" name="showContactTable">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>90</y>
             <width>481</width>
             <height>201</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="showLabel">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>30</y>
             <width>181</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>AppleMyungjo</family>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Show Contacts</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGraphical_Phonebook">
    <property name="title">
     <string>GraphicalPhonebook</string>
    </property>
   </widget>
   <addaction name="menuGraphical_Phonebook"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
